auto_landing/src/tracking.cpp

```diff
@@ -0,0 +1,93 @@
+#include<auto_landing/tracking.hpp>
+
+namespace ariitk::auto_landing {
+
+Tracking::Tracking() :
+    height_(1) {}
+
+void Tracking::init(ros::NodeHandle& nh, char** argv) {
+    char* end_ptr;
+    husky_relative_x_ = strtod(argv[2], &end_ptr);
+    husky_relative_y_ = strtod(argv[4], &end_ptr);
+
+    ROS_INFO ("Husky spawned at %lf %lf", husky_relative_x_, husky_relative_y_);
+
+    set_firefly_pose_ = nh.advertise<geometry_msgs::PoseStamped>("commad_pose", 1);
+    quad_pose_ = nh.subscribe("quad_odometry", 1, &Tracking::quadPoseCallback, this);
+    gazebo_model_state_=nh.subscribe("model_state",1,&Tracking::modelStateCallback,this);
+    landing_client_ = nh.serviceClient<std_srvs::Trigger>("to_land");
+}
+
+void Tracking::run(){
+    landing_service_.response.success=false;
```

Comment:
setting response is of no use. response is returned by the service call. if you want set something, set the request fields. otherwise, just remove this line